### DISCUSSION TITLE
Strange behavior in quotes

### DISCUSSION THREAD

**Discussion by cesarreyes3** [haidubogdan/netbeans-php-blade-plugin#69] [Q&A]
```blade
<script>
function test(){
    method('test);
}
</script>
```
Copy previous code on a blade file, and type the missing `'`, netbeans writes the `'` twice,
Is this expected behavior or a blade bug?
![image](https://github.com/user-attachments/assets/e02b228b-90e7-4cb8-851b-ca25139a54ca)

Also if you backspace the first one that appeared, both are deleted

**Comment by haidubogdan** [comment]
I've missed this one.
I didn't reproduce it so far.
The automatic quotes completion of quotes might come from netbeans ide, as the only quick autocompletion I've added are for brackets and blade tags.
